webpack.config.js

```diff
@@ -85,13 +85,6 @@ const webConfig = {
 			test: /\.ts$/,
 			exclude: /node_modules/,
 			use: [{
-                // vscode-nls-dev loader:
-                // * rewrite nls-calls
-                loader: 'vscode-nls-dev/lib/webpack-loader',
```

Comment:
Have you tested if removing this breaks localization on the web extension?

To check that localization occurs correctly,

1. Add a translation for one of your strings in one of the `loc/translations-import/vscode-powerplatform.*.xlf` files.  Make sure this is obviously not an official translation with text along the lines of "TESTING LOCALIZATION LOADING" or whatnot
2. run the compilation steps to generate the localized artifacts
3. install the VS Code language pack extension correlating to the language you added the fake translation for
4. switch to that language
5. debug launch your extension, and navigate to where that string appears.   Is the fake translation displayed?

Reply:
This was removed because webExtension doesn't support localization at the moment. There is an open bug: https://github.com/microsoft/vscode/issues/82595 

It is expected to be fixed by end of this month. So in the next release we can check on this again.